docs/acceptance-criteria/uacs-tbtc-v2.adoc

```diff
@@ -0,0 +1,291 @@
+| If the DKG timeout notification was deemed justified, then: +
+  - the pool should be unlocked +
+  - the fixed amount reward (in T) (`sortitionPoolUnlockingReward`) should be
+  sent from the DAO-funded ETH pool to the notifier.
+
+// TODO: add UACs for hearbeats, wallet closure, sweeping, deposits, redemptions, etc
+
+| Governance - group size
+| The number of members selected to the candidate signing group (`group_size`)
+  should NOT be a governable parameter and should be set to `100`.
+
+| Governance - group size
+| The minimum number of members we’re allowed to drop down to during the DKG
+  group formation re-try period (`dkg`) should be a governable parameter and
+  should be initially set to `90`.
+
+| Governance - wallet lifecycle
+| The minimum time since creation of the last wallet after which a new wallet
+  can be created (`wallet_creation_period`) should be a governable parameter and
+  should be initially set to `1 week`.
+
+| Governance - wallet lifecycle
+| The smallest amount of btc a wallet can hold before we attempt to close the
+  wallet and transfer the funds to a randomly selected wallet (`wallet_min_btc`)
+  should be a governable parameter and should be initially set to ...
+
+// This UAC is based on the assumption that that the result submission
+// conditions will be the same as for Random Beacon (at first one operator can
+// submit, after `resultSubmissionEligibilityDelay` another is added to the list
+// of operators eligible to publish the result, and so on...). This is how
+// initial implementation of a DKG fo ECDSA looks like, but there are ongoing
+// discussions wheter this is the best approach. We may end up not having
+// `resultSubmissionEligibilityDelay` param.
+| Governance - DKG
+| The frequency (in blocks) of adding new group members as eligible to submit a
+  DKG result (`resultSubmissionEligibilityDelay`) should be a governable
+  parameter. Its initial value should be set to `20 blocks`.
+
+| Governance - DKG
+| The reward for notifying about DKG timeot (`sortitionPoolUnlockingReward`)
+  should be a governable parameter and should be initially set to ....
+
+| Governance - DKG
+| Percentage of the amount slashed from the group as a result of signing a
+  malicious DKG result, transferred as a reward to the notifier,
+  (`dkgMaliciousResultNotificationRewardMultiplier`) should be a governable
+  parameter. Its initial value should be set to `100`.
+
+| Governance - DKG
+| The slashing amount for submitting malicious DKG result
+(`maliciousDkgResultSlashingAmount`) should be a governable parameter and should
+be initially set to `50000e18` (50000 T).
+
+| Governance - sortition pool
+| The length of the period during which operators won't be able to earn rewards
+  as a punishment for misbehavior during submission of DKG result
+  (`sortitionPoolRewardsBanDuration`) should be a governable parameter. Its
+  initial value should be set to `2 weeks`.
```

Comment:
Is using a reward ban something that we go with? There are ongoing discussions how to handle this so that the cost of banning somebody would not be higher than savings made by banning him. With the current idea we also need to figure out how to correctly prolongate a ban on an operator which already is banned.